Formula/openjdk.rb

```diff
@@ -1,10 +1,8 @@
 class Openjdk < Formula
   desc "Development kit for the Java programming language"
   homepage "https://openjdk.java.net/"
-  url "https://hg.openjdk.java.net/jdk-updates/jdk13u/archive/jdk-13.0.2+8.tar.bz2"
-  version "13.0.2+8"
-  sha256 "01059532335fefc5e0e7a23cc79eeb1dc6fea477606981b89f259aa0e0f9abc1"
-  revision 2
+  url "https://hg.openjdk.java.net/jdk/jdk/archive/jdk-14+36.tar.bz2"
+  version "14"
```

Comment:
Should we follow previous format? `14+36`

Reply:
This causes an audit failure due to the version parsing code

```console
% brew audit --strict openjdk
openjdk:
  * Formula has other versions so create a versioned alias:
      cd /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Aliases
      ln -s ../Formula/openjdk.rb openjdk@14+36
  * Formula has invalid versioned aliases:
      openjdk@14
```

Reply:
It's also not clear to me whether we want to package patch versions of openjdk. It seems sensible to only package the *-ga versions, but looking at Debian it seems that they _do_ package patch versions so 🤷‍♂

Reply:
Looks like this will need the `14.0+36` version since I don't see a way to determine the build number without a flag. Needs to be 14.0 and not just 14 due to the version parsing issue w.r.t aliases as above.

Reply:
I'd say let's stick to only the `*-ga` versions, like AdoptOpenJDK. Maybe we should just use `14.0.0+36` for the version, since the pattern will be the same for the for the `14.0.1+x` version.